SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Supported Versions
+
+Redis is generally backwards compatible with very few exceptions, so we
+recommend users to always use the latest version to experience stability,
+performance and security.
+
+We generally backport security issues to a single previous major version,
+unless this is not possible or feasible with a reasonable effort.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 7.2.x   | :white_check_mark: |
+| 7.0.x   | :white_check_mark: |
+| 6.2.x   | :white_check_mark: |
+| < 6.2   | :x:                |
+
+## Reporting a Vulnerability
+
+If you believe you've discovered a serious vulnerability, please contact the
+Redis core team at redis@redis.io. We will evaluate your report and if
+necessary issue a fix and an advisory. If the issue was previously undisclosed,
+we'll also mention your name in the credits.
```

Comment:
don't we want the community to report with a ticket as well?

Reply:
It supposed, after have the policy in place, to add one button to the community report it